controlplane/build/Dockerfile.nsmd

```diff
@@ -1,8 +1,9 @@
-FROM golang:alpine as build
-
 ARG VERSION=unspecified
 ARG VENDORING
 ARG GOPROXY
+ARG GO_VERSION
+
+FROM golang:${GO_VERSION}-alpine as build
```

Comment:
How about extracting the whole 'golang:${GO_VERSION}-alpine' thing as an ARG somethinglike BASE_IMAGE?

Reply:
I'm fine with just go version to be passed.